src/Tasks/UpdatePackageInfoTask.php

```diff
@@ -213,15 +216,22 @@ public function getPackageInfo($packageList)
     }
 
     /**
-     * Return an array of supported modules as fetched from addons.silverstripe.org. Outputs a message and returns null
-     * if an error occurs
+     * Return an array of supported modules as fetched from silverstripe/supported-modules.
+     * Outputs a message and returns null if an error occurs
      *
      * @return null|array
      */
     public function getSupportedPackages()
     {
         try {
-            return $this->getSupportedAddonsLoader()->getAddonNames() ?: [];
+            $repos = MetaData::getAllRepositoryMetaData()[MetaData::CATEGORY_SUPPORTED];
+            $version = VersionProvider::singleton()->getModuleVersion('silverstripe/framework');
+            preg_match('/^([0-9]+)/', $version, $matches);
+            $cmsMajor = BranchLogic::getCmsMajor(MetaData::getMetaDataForRepository('silverstripe/silverstripe-framework'), $matches[1] ?? '');
+            return array_filter(array_map(
+                fn(array $item) => isset($item['majorVersionMapping'][$cmsMajor]) ? $item['packagist'] : null,
+                $repos
+            ));
```

Comment:
Note this is not hardcoded to `5` anymore, to reduce the changes of it being stale if this module gets a CMS 6 compatible release.